seed/energy/meter_data_processor/green_button_parser.py

```diff
@@ -0,0 +1,167 @@
+from seed.energy.meter_data_processor import green_button_maps
+from datetime import date
+
+global root_ns
+global gb_ns
+
+root_ns = '{http://www.w3.org/2005/Atom}'
+gb_ns = '{http://naesb.org/espi}'
+
+def get_link_href_head(link):
+    href = link.get('href')
+    if href[0:1]=='/':
+        href = href[1:]
+    first_slash = href.find('/')
+    return href[0:first_slash]
+
+def get_link_href_tail(link):
+    href = link.get('href')
+    last_slash = href.rfind('/')
+    return href[(last_slash+1):]
+
+def parse_usage(up_type):
```

Comment:
The format is required by GreenButton, a valid GreenButton XML file should have these nodes. If the nodes are missing, it will let caller know the XML is not valid.